.github/workflows/changelog.yml

```
@@ -6,6 +6,7 @@ on:
       - 'release/**'
   pull_request:
     paths:
+      - .changelog/
```

Reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-file-paths

```suggestion
      - .changelog/*
```